spring-cloud-stream-reactive/src/test/java/org/springframework/cloud/stream/reactive/StreamListenerReactorTests.java

```diff
@@ -165,7 +334,17 @@ public void receive(@Input(Processor.INPUT) Flux<String> input, @Output(Processo
 
 	@EnableBinding(Processor.class)
 	@EnableAutoConfiguration
-	public static class TestInputOutputArgsWithMessage {
+	public static class TestInputOutputArgs2 {
+
+		@StreamListener(Processor.INPUT)
```

Comment:
ok, that would simplify things. My intention was to support all possible combinations of method annotations and parameter annotations.